apps/src/sites/studio/pages/blocks/edit.js

```diff
@@ -11,7 +11,13 @@ import { installCustomBlocks } from '@cdo/apps/gamelab/blocks';
 let nameField;
 $(document).ready(() => {
   nameField = document.getElementById('block_name');
-  Blockly.inject(document.getElementById('blockly-container'), {});
+  Blockly.inject(document.getElementById('blockly-container'), {
+    valueTypeTabShapeMap: {
+      Sprite: 'angle',
+      Behavior: 'rounded',
+      Location: 'square',
```

Comment:
these key names are `Blockly.BlockValueType` values, right? Could we reference them as such? I don't love having capitalized keys without making clear the reason why

Reply:
Sounds good. I also extracted a constant so that this can be shared between spritelab levels and the block editor.